tests/modules/configurations/resources/utils.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+"""
+Configuration resources utils
+-------------
+"""
+from tests import utils as test_utils
+import json
+
+EXPECTED_KEYS = {'response', 'success'}
+CONFIG_PATH = '/api/v1/configuration/default'
+CONFIG_DEF_PATH = '/api/v1/configurationDefinition/default'
+
+
+def read_configuration(
```

Comment:
Good step to factor these out but you should go one step further and use the get_dict_via_flask etc functions for these.
https://github.com/WildMeOrg/houston/blob/main/tests/utils.py#L235
See https://github.com/WildMeOrg/houston/blob/main/tests/modules/notifications/resources/utils.py for examples of how to do this.